twitch-rate/src/components/modal/modal.ts

```diff
@@ -1,12 +1,12 @@
-import { modalStore } from '@skeletonlabs/skeleton';
 
-export const modal: ModalSettings = {
+import { Modal, modalStore } from '@skeletonlabs/skeleton';
+import type { ModalSettings, ModalComponent } from '@skeletonlabs/skeleton';
```

Comment:
line 2's imports don't seem to be in use, and `ModalComponent` is also not being used, so we can remove those.

Reply:
next thing we'll do is get the modal to appear in the middle of the screen like the examples from https://www.skeleton.dev/utilities/modals